soc/st/stm32/stm32wbax/hci_if/linklayer_plat_local.h

```diff
@@ -8,6 +8,6 @@
 #ifndef _STM32WBA_LINK_LAYER_PLAT_LOCAL_H_
 #define _STM32WBA_LINK_LAYER_PLAT_LOCAL_H_
 
-void link_layer_register_isr(void);
+void link_layer_register_isr(bool force);
```

Comment:
An line description comment would be nice to explicitly tell what `force` is about.